docs/cli_tools.rst

```diff
@@ -86,53 +86,109 @@ Example
     Time Remaining: 104
     Wattage: -41
 
-pi-top brightness
+pi-top display
 =========================
 
-On pi-top devices with a screen, it allows to query and control its brightness.
-
-Running `pt-brightness` on its own will report back the current brightness value.
+On pi-top devices with a screen, it allows to control different display settings.
 
 .. code-block:: bash
 
-    pi-top brightness [-h] [-b {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16}] [-i]
-                     [-d] [-l {0,1}] [-t TIMEOUT] [-v]
+    pi-top display [-h] {brightness,backlight,timeout}
+
+Where:
+
+-h, --help
+    Show a help message and exits
+
+brightness
+    Control display brightness
+
+backlight
+    Control display backlight
+
+timeout
+    Set the timeout before the screen blanks in seconds (0 to disable)
+
+
+pi-top display brightness
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Using `pi-top display brightness` without arguments will return the current brightness value.
+
+.. code-block:: bash
 
+    pi-top display brightness [-h] [-v] [-i] [-d]
+                                 [brightness_value]
 
 Where:
 
 -h, --help
     Show a help message and exits
 
--b {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16}, --brightness_value {1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16}
-    Set screen brightness level [1-10] on pi-topHUB, or
-    [1-16] or pi-topHUB v2
+-v, --verbose
+    Increase verbosity of output
 
 -i, --increment_brightness
     Increment screen brightness level
 
 -d, --decrement_brightness
     Decrement screen brightness level
 
--l {0,1}, --backlight {0,1}
-    Set the screen backlight state [0-1]
-
--t TIMEOUT, --timeout TIMEOUT
-    Set the timeout before the screen blanks in seconds (0
-    to disable)
-
--v, --verbose
-    Increase output verbosity
+brightness_value
+    Set screen brightness level [1-10] on pi-topHUB, or
+    [1-16] or pi-topHUB v2
 
 
 Example
 ~~~~~~~~~~~~~~~~~
 
 .. code-block:: bash
 
-    pi@pi-top:~ $ pi-top brightness
+    pi@pi-top:~ $ pi-top display brightness
     16
 
+
+pi-top display brightness
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
Request or change the value of the display's brightness.

Note: this only works for the original pi-top, pi-topCEED and pi-top [3]. The pi-top [4] FHD Touch Display uses hardware buttons to control the brightness, and is not controllable via this SDK.

Reply:
Also explain that pi-top [3] is 0-16, pi-top [1] and CEED is 0-10.